src/sentry/integrations/source_code_management/commit_context.py

```diff
@@ -352,6 +352,36 @@ def queue_comment_task_if_needed(
 
                     pr_comment_workflow.queue_task(pr=pr, project_id=group_owner.project_id)
 
+    def queue_open_pr_comment_task_if_needed(
+        self, pr: PullRequest, organization: Organization
+    ) -> None:
+        try:
+            open_pr_comment_workflow = self.get_open_pr_comment_workflow()
```

Comment:
For GitHub Enterprise, this raises a NotImplementedError, which is caught and returned below.